### PR TITLE
sys-cluster/spark-bin: bump from 2.4.3 to 2.4.4

### PR DESCRIPTION
**Pull request by trozamon**

### CODE REVIEW

**Comment by juippis** [comment]
Noo don't use `git mv` but rather split addition and removal into their separate commits.

**Comment by trozamon** [comment]
> Noo don't use `git mv` but rather split addition and removal into their separate commits.

Ah, ok, I'll remember that for the future. How do the new commits look?

File: sys-cluster/spark-bin/spark-bin-2.4.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline sys-cluster/spark-bin/spark-bin-2.4.4.ebuild]: They look much better! Could you however try building it with EAPI-7 updated here? Namely the eclass should support it...

Comment (trozamon) [reply sys-cluster/spark-bin/spark-bin-2.4.4.ebuild]: Done - working fine for me. Thank you for all the review.

**Comment by juippis** [comment]
Thanks!

**Pull request closed without merging**